### PR TITLE
[Peterborough] Add tree layers from EzyTreev

### PR DESCRIPTION
**Pull request by chrismytton**
Use the data we're getting from Peterborough's SFTP server (see https://github.com/mysociety/tilma/pull/6) to add a couple of tree layers, one for individual trees and one for groups of trees.

### CODE REVIEW

File: layers/peterborough.map [hunk 1]
```diff
@@ -2,6 +2,7 @@ MAP
   NAME "Peterborough"
   STATUS ON
   EXTENT -180 -90 180 90
+  SHAPEPATH "../../layers"
```
Comment (dracos) [inline layers/peterborough.map]: If this is added, I guess it should (needs to? not sure) be removed from the existing highways layer.

**Review by dracos — CHANGES_REQUESTED** [review]
One small suggestion is all :)
As this is in progress work, generally what we do is merge this to a 'peterborough-reviewed' branch or similar, which in turn is then merged to staging for deploy on there, until such time as it's approved/working and then it's merged to master. And then any changes you need to make can be reviewed on top of that reviewed branch and (probably in this case) all squashed together before merge. See e.g. my Bexley trees which they've just changed the asset ID field again.
As this is a tiny repo, doesn't matter so much but with the equivalent code at the FMS end, for example.

File: layers/peterborough.map (hunk 1 above)
Comment (chrismytton) [reply layers/peterborough.map]: I wasn't sure if `SHAPEPATH` was specific to `DATA`, but I've checked and `CONNECTION` works either way, so I've changed it to the short version in 8af282484f69c4f1ebd6874a4874a639516fd469.

**Review by dracos — APPROVED** [review]
🌲

**Comment by chrismytton** [comment]
Oops, I messed up the merge so this PR is marked as closed rather than merged, but don't worry, it has been merged into the `peterborough-reviewed` branch.

**Pull request closed without merging**